### ISSUE TITLE
Multiple possible approvers, approver groups

### ISSUE THREAD

**Issue by ltning** [gkrid/dokuwiki-plugin-approve#21]
It would be nice if groups could be selected as eligble approvers.

In addition, if specifying the same namespace/page name several time with different approvers, all of them should be able to approve the page (not just the last one in the list, as it currently seem to work)

**Comment by solewniczak** [comment]
Yes, I think the plugin should be extended to support multiple approvers (and groups). I'll try to extend the plugin with it on some occasion.

**Comment by jplitza** [comment]
> It would be nice if groups could be selected as eligble approvers.

Apparently, this is possible in theory, but doesn't work with LDAP. At least that's what I read from #32 

> In addition, if specifying the same namespace/page name several time with different approvers, *all* of them should be able to approve the page (not just the last one in the list, as it currently seem to work)

I probably should open a different issue, but I would like to have an option that *every* group has to approve. If I wanted any, I could just create a new group containing the union of all allowed groups.
